### PR TITLE
Added `operationName`

### PR DESCRIPTION
**Pull request by OlegIlyenko**

### CODE REVIEW

File: site/docs/BestPractice-ServingOverHTTP.md [hunk 1]
```diff
@@ -44,7 +44,8 @@ A standard GraphQL POST request should use the `application/json` content type,
 ```js
 {
   "query": "...",
-  "variables": { variable1: value, ... }
+  "oprationName": "...",
```
Comment (stubailo) [inline site/docs/BestPractice-ServingOverHTTP.md]: I think you're missing the `e` in `operationName` here and also above.

Also, I think it's worth mentioning that the `operationName` in the POST body is optional, and only required if you have multiple operations?

As an additional wrinkle, express-graphql doesn't support multiple operations in one document as far as I know.

Comment (OlegIlyenko) [inline site/docs/BestPractice-ServingOverHTTP.md]: OMG, sorry for the typo :flushed:

I was also thinking about adding a note about optional fields. In this case we need to mention that `variables` is optional as well. WDYT?

Comment (stubailo) [inline site/docs/BestPractice-ServingOverHTTP.md]: Yeah, maybe just add a sentence underneath the code snippet saying operationName and variables are optional.

Comment (OlegIlyenko) [inline site/docs/BestPractice-ServingOverHTTP.md]: Perfect. I already added it and rebased the PR

**Comment by stubailo** [comment]
Just one last question - should we be worried that `express-graphql` doesn't support this feature?

**Comment by OlegIlyenko** [comment]
Are you sure about this limitation? I just double checked the docs and looks like `operationName` is supported: https://github.com/graphql/express-graphql#http-usage

**Comment by stubailo** [comment]
Oh, OK I'm totally wrong :]

**Pull request merged**